### DISCUSSION TITLE
after updating to  Xcode 16

### DISCUSSION THREAD

**Discussion by AdwaidSuresh001** [pichillilorenzo/flutter_inappwebview#2357] [Q&A]
Failed to build iOS app
Error output from Xcode build:
↳
--- xcodebuild: WARNING: Using the first of multiple matching destinations:
2
{ platform:iOS Simulator, id:5D168DF6-7EE9-4AEE-91D7-CC9B0B5EF3FE, OS:17.5, name:iPhone 15 Pro Max }
** BUILD FAILED **
![Screenshot 2024-10-18 at 3 51 33 PM](https://github.com/user-attachments/assets/ed661e0f-8798-420c-b6a3-39e7a3c3fe78)

**Comment by pichillilorenzo** [comment]
You should update to the latest flutter plugin
